src/server/server_family.cc

```diff
@@ -2810,6 +2810,7 @@ string ServerFamily::FormatInfoMetrics(const Metrics& m, std::string_view sectio
   if (should_enter("CLUSTER")) {
     append("cluster_enabled", IsClusterEnabledOrEmulated());
     append("migration_errors_total", service_.cluster_family().MigrationsErrorsCount());
+    append("migrated_keys_total", service_.cluster_family().MigratedKeysCount());
```

Comment:
both migration_errors_total and migrated_keys_total take migration mutex.
Could we unit them to single function so we will take the mutex once?

Reply:
i think the whole approach should be aligned to what we do with other components - setup a thread local struct and fetch data from it into metrics struct. We do it with `tl_facade_stats` for example. Here we could use `EngineShard::Stats`